src/components/navbars/MobileNav.jsx

```diff
@@ -0,0 +1,15 @@
+import joinIcon from '../../assets/icons/register.png';
+import loginIcon from '../../assets/icons/login.png';
+import searchIcon from '../../assets/icons/search.png';
+
+const MobNav = ({onSearch, onLogin, onJoin, stateSearch, stateLogin, stateJoin}) => {
+    return(
+        <nav className="sm:hidden relative flex flex-row my-4 gap-2">
+            { stateSearch && <button onClick={onSearch} aria-label="Search"><img src={searchIcon} className="h-6" alt="search icon"/></button> }
+            { stateLogin && <button onClick={onLogin} aria-label="Login"><img src={loginIcon} className="h-6" alt="login icon"/></button> }
```

Comment:
Would it be a good idea to add title attributes to buttons?
For example, `title="Search"` for the first button? So that users can see it when hovering the button? Like in the image below

![title-demo](https://github.com/user-attachments/assets/c8c01e43-f4c1-453c-8b12-3e89711cef61)

Reply:
I rarely see such thing, I'm not sure, that the user will see it on the mobile version.

Reply:
You are right